cabal-docspec/cabal-docspec.1

```diff
@@ -575,6 +575,98 @@ Failures in the setup code does not cause module skip.
 .PP
 GHC-7.0 relies that \f[I]Char\f[R] type is in scope.
 This is an implementation artifact.
+.SH Q&A
+.SS Q: cabal-docspec doesn\[cq]t import modules automatically
+.PP
+cabal-docspec tests library documentation from the outside.
```

Comment:
I presume that this information is exactly the same as what is contained in `MANUAL.md`? I won't look at this closely, but let me know if there is any additional information here.

Reply:
Yes. This file is generated from `MANUAL.md`.